app/jobs/index_job.rb

```diff
@@ -59,6 +59,7 @@ def perform(file_paths)
 
     logger.info("Requesting a batch Solr update...")
     blacklight_connection.add(solr_documents)
+    blacklight_connection.commit
```

Comment:
In general we've moved away from committing as part of application code. We have soft and hard commits configured in the solrconfig (https://github.com/pulibrary/pul_solr/blob/352c072e0ab2e4913f39912d1704b1d8f93c9ab6/solr_configs/pulfalight-staging/conf/solrconfig.xml#L316-L325). Should we revisit this decision?

Reply:
No, I am not interested in moving away from the solrconfig-based approach. I was finding that I needed to `curl` in order to commit Documents which were indexed with Rake tasks, and I assumed that this was needed.